Kitodo/src/test/resources/log4j2.xml

```diff
@@ -21,6 +21,9 @@
         <Logger name="de.unigoettingen.sub.search.opac, org.goobi, org.kitodo" level="debug">
             <AppenderRef ref="STDOUT"/>
         </Logger>
+        <Logger name="org.kitodo.production.process" level="trace" additivity="false">
```

Comment:
Is this neccessary for the test?

Reply:
Yes, Travis does not complete otherwise.